src/Middleware/RequestBodyParserMiddleware.php

```diff
@@ -9,11 +9,20 @@
     public function __invoke(ServerRequestInterface $request, $next)
     {
         $type = strtolower($request->getHeaderLine('Content-Type'));
+        list ($type) = explode(';', $type);
 
         if ($type === 'application/x-www-form-urlencoded') {
             return $next($this->parseFormUrlencoded($request));
         }
 
+        if ($type === 'multipart/form-data') {
+            return $next(MultipartParser::parseRequest($request));
+        }
+
+        if ($type === 'multipart/mixed') {
+            return $next(MultipartParser::parseRequest($request));
+        }
```

Comment:
Couldn't find a *relevant* mention of `multipart/mixed` in https://tools.ietf.org/html/rfc7578#section-4.3 and https://www.w3.org/TR/html5/forms.html#multipart-form-data, is this needed?

Reply:
IIRC I've seen both during development, I'll run some tests when I get home to be sure of this 👍